modules/virt-about-vtpm-devices.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-using-vtpm-devices.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-vtpm-devices_{context}"]
+= About vTPM devices
+
+A Virtual Trusted Platform Module (vTPM) device functions like a
+physical Trusted Platform Module (TPM) hardware chip.
+
+You can use a vTPM device with any operating system, but Windows 11 requires
+the presence of a TPM chip to install or boot. A vTPM device allows VMs created
+from a Windows 11 image to function without a physical TPM chip.
+
+If you do not enable vTPM, then the VM does not recognize a TPM device, even if
+the node has one.
+
+vTPM devices also protect virtual machines by temporarily storing secrets
+without physical hardware. However, vTPM discards stored secrets after each VM
```

Comment:
I think it's worth mentioning here why the vTPM discard stored secrets here (because like Jed mentioned before we do not yet support state persistence)